R/s3.put.R

```diff
@@ -7,7 +7,7 @@
 #' @rdname s3.get
 s3.put <- function (x, path, name, bucket.location = "US", verbose = FALSE,
                     debug = FALSE, encrypt = FALSE, check_exists = TRUE,
-                    num_retries = getOption("s3mpi.num_retries", 0)) {
+                    num_retries = getOption("s3mpi.num_retries", 0), backoff = c(2,8,32,64,128)) {
```

Comment:
Yeah, I was looking at just doing an exponential, but we'd still have to cap it (otherwise by 2^10 we in crazytown) and the splits for 3^x seem a little more natural.  I can exponentiate and cap.